### PR TITLE
Fixed height of Privacy Policy modal #186

### LINKED ISSUE #186: Fix height of Privacy Policy modal
Privacy policy modal has whitespace.  It should be fit to the content size.

image

### PR DESCRIPTION
**Pull request by feysalikbal**
This PR intends to fix the issue of extra white space in the Privacy Policy modal.

## [Screenshots]
privacypolicymodal
 

Closes #186

### CODE REVIEW

File: src/pages/home/components/LegalModal.scss [hunk 1]
```diff
@@ -2,3 +2,7 @@
 .panelRoot {
   height: 100%;
 }
+
+.panelPrivacy {
+  height: 77%;
```
Comment (tureey) [inline src/pages/home/components/LegalModal.scss]: Why this number? 🤔

Comment (feysalikbal) [reply src/pages/home/components/LegalModal.scss]: .innerModal in modal.scss has 100% height. So, If there is less content it is still displayed without adjusting the height itself. That's why the whitespace occurs.

**Comment by crysfel** [comment]
@tureey are we good with this? Should I merge or your requested changes still need to be done? I'm not clear what are those request changes, maybe @feysalikbal neither  😂

**Comment by tureey** [comment]
@crysfel Download the PR and try it on mobile. Try all the modals (footer buttons) And tell me if you see them OK (try to scroll though) Thank you! 🙏

**Comment by tureey** [comment]
ping @crysfel

**Comment by crysfel** [comment]
@feysalikbal could you make sure it works well in mobile and desktop? Thanks!

**Comment by emmawedekind** [comment]
Hey - what's the status here!

**Comment by crysfel** [comment]
Closing this

**Pull request closed without merging**